app/temporal/src/domain/DataAnalyzer.spec.ts

```diff
@@ -103,11 +110,10 @@ describe("DataAnalyzer", () => {
         { __rowId: 2, age: 25 },
         { __rowId: 3, name: "" },
         { __rowId: 4, name: null },
-        { __rowId: 5, name: undefined },
```

Comment:
was ist an dem testcase verkehrt?

Reply:
Undefined ist aktuell nicht in den erlaubten Types. Eigentlich müsste man null auch entfernen, da leere Zellen mit "" aufgefüllt werden